src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { Person } from '../types/Person';
+import { peopleFromServer } from '../data/people';
+import debounce from 'lodash.debounce';
+
+type Props = {
+  onSelected?: (person: Person | null) => void;
+  setSelectedPerson: (person: Person) => void;
+  delay?: number;
+};
+
+export const Autocomplete: React.FC<Props> = ({
+  onSelected,
+  setSelectedPerson,
+  delay = 300,
+}) => {
+  const [query, setQuery] = useState<string>('');
+  const [appliedQuery, setAppliedQuery] = useState<string>('');
+  const [isDropdownOpen, setIsDropdownOpen] = useState<boolean>(false);
+
+  const applyQuery = useCallback(debounce(setAppliedQuery, delay), []);
```

Comment:
Fix this linter error 
<img width="992" alt="image" src="https://github.com/user-attachments/assets/dd71a59f-1a1f-472e-bb86-a0f7cc94aee8">